pkg/build/controller/buildconfig_controller.go

```diff
@@ -35,14 +36,21 @@ func IsFatal(err error) bool {
 
 type BuildConfigController struct {
 	BuildConfigInstantiator buildclient.BuildConfigInstantiator
-
+	BuildConfigGetter       buildclient.BuildConfigGetter
+	BuildLister             buildclient.BuildLister
+	BuildDeleter            buildclient.BuildDeleter
 	// recorder is used to record events.
 	Recorder record.EventRecorder
 }
 
 func (c *BuildConfigController) HandleBuildConfig(bc *buildapi.BuildConfig) error {
 	glog.V(4).Infof("Handling BuildConfig %s/%s", bc.Namespace, bc.Name)
 
+	glog.V(4).Infof("Handling Build Pruning: %v/%v", bc.Namespace, bc.Name)
```

Comment:
this logging is redundant, it doesn't add any information to what was logged above.